docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>"This tutorial presents how to run custom operations written as a Numba JIT-compiled function."</p><p> </p><p>The second sentence doesn't apply anymore, since the user is not providing a cfunc anymore. How about:</p><p>"The operator translates the user function into a compiled function callable from foreign C code, eliminating the Python interpreter overhead."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>